### PR TITLE
4.get api reviews

### PR DESCRIPTION
**Pull request by Kristaps31**
Please see completed task 4.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const experss = require("express");
+const {getCategories, getReviews} = require("./controllers/games.controllers")
+const app = experss();
```
Comment (Beth-McCormack) [inline app.js]: check spelling of express

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        return request(app)
+        .get('/api/categories')
+        .expect(200)
+        .then(({ body }) => {
+            const {categories} = body;
+            expect(categories).toHaveLength(4);
+            expect(categories[0]).toHaveProperty("slug", expect.any(String));
+            expect(categories[0]).toHaveProperty("description", expect.any(String));
+        })
+    })
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: This is only testing the first element in the array. Use iteration to check each element is what you'd expect.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                expect(review).toHaveProperty("designer", expect.any(String));
+                expect(review).toHaveProperty("owner", expect.any(String));
+                expect(review).toHaveProperty("review_img_url", expect.any(String));
+                expect(review).toHaveProperty("review_body", expect.any(String));
+                expect(review).toHaveProperty("category", expect.any(String));
+                expect(review).toHaveProperty("created_at", expect.any(String));
+                expect(review).toHaveProperty("votes", expect.any(Number));
+            })
+        });
+    })
```
Comment (Beth-McCormack) [inline __tests__/app.test.js:68]: can you add an expect statement here to check that the reviews are sorted in descending order as per the ticket description

File: __tests__/app.test.js (hunk 2 above)
Comment (Beth-McCormack) [reply __tests__/app.test.js:27]: This is only testing first element in the array. Use iteration to check each element is what you'd expect.

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

**Review by johnhutcheon — APPROVED** [review]
As per our slack conversation, I'll approve this, but just look at using Jest Sort in future to make things easier!

**Pull request merged**